Directory.Build.targets

```diff
@@ -84,9 +84,9 @@
     <PackageReference Update="Microsoft.Extensions.Logging.Debug" Version="$(ExtensionsVersion)" />
     <PackageReference Update="Microsoft.Extensions.Options" Version="$(ExtensionsVersion)" />
     <PackageReference Update="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(ExtensionsVersion)" />
-    <!-- Warning: as of 2021.01.19, the rest of `Extension` packages is at version 5.0.0 and these two are at 5.0.1 and 5.0.2 respectively :) -->
-    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="5.0.1" />
```

Comment:
This is now outdated comment :P

Reply:
Which is why it's gone now?

Reply:
Yeah, but the new one is also irrelevant :p

Reply:
I'd say that if attempting to use a `Microsoft.Extensions.*` package with version `$(ExtensionsVersion)` results in an error, the comment is just as justified as the previous one, otherwise this small discrepancy would be too easy to overlook.

Reply:
I'll try to move it around so it doesn't blend with other `Microsoft.Extension.*` packages, but I'd let the comment stay anyway, at least until .NET 6 becomes GA and these `-preview…` suffixes go away.